src/status_im/translations/fr.cljs

```diff
@@ -287,9 +300,14 @@
    :wallet-add-asset                      "Ajouter un actif"
    :wallet-total-value                    "Valeur totale"
    :wallet-settings                       "Paramètres du portefeuille"
+   :wallet-manage-assets                  "Gérer les actifs"
    :signing-phrase-description            "Signez la transaction en saisissant votre mot de passe. Assurez-vous que les mots ci-dessus correspondent à votre phrase de signature secrète"
    :wallet-insufficient-funds             "Fonds insuffisants"
    :request-transaction                   "Effectuer une transaction"
+   :receive-transaction                   "Recevoir une transaction"
+   :new-transaction                       "Nouvelle Transaction"
```

Comment:
What's the difference between a "transaction request" and a "new transaction" in English?

Reply:
Not used in same context. Action vs title